client/rest/.gitignore

```diff
@@ -0,0 +1 @@
+/build-eclipse-default/
```

Comment:
Should this be in root?

Reply:
Probably will be good to drop it for now, sorry for that, planning to complement DEVELOPER.md with Eclipse instructions